java/23ai-springboot3-jpa-ucp/pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>3.3.0</version>
+    <relativePath /> <!-- lookup parent from repository -->
+  </parent>
+
+  <groupId>com.oracle.dev.jdbc</groupId>
+  <artifactId>23ai-springboot3-jpa-ucp</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>23ai-springboot3-jpa-ucp</name>
+  <description>Java Persistence with Spring Boot 3, Spring Data JPA with
+    Hibernate, and the Oracle Database 23ai Free</description>
+  <url>
+    https://juarezjunior.medium.com/java-persistence-with-spring-boot-3-spring-data-jpa-with-hibernate-and-the-oracle-database-23c-9c0946d4838f</url>
+  <properties>
+    <java.version>21</java.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>21</maven.compiler.source>
+    <maven.compiler.target>21</maven.compiler.target>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-data-jpa</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-web</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-test</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.oracle.database.jdbc</groupId>
+      <artifactId>ojdbc11-production</artifactId>
+      <version>23.4.0.24.05</version>
```

Comment:
Please upgrade to 23.6.

Reply:
@Kuassim I have followed Jean's suggestion and updated the driver version to 23.6.0.24.10. 
However, let's try to abide by our latest agreements, as asking recurrently to update the driver version before the very first release of a given code sample configures a dead-end situation, where it's being kept in a situation of open modification. 
Such updates (versions/releases) will be done as part of our agreed Quarterly QA Freshness Pass activity. 
@Kuassim please approve.